clang/test/SemaCXX/attr-mode-tmpl.cpp

```diff
@@ -9,7 +9,7 @@ void CheckEnumerations() {
   // Check that non-vector 'mode' attribute is OK with enumeration types.
   typedef T __attribute__((mode(QI))) T1;
   typedef T T2 __attribute__((mode(HI)));
-  typedef T __attribute__((mode(V8SI))) T3; // expected-error{{mode 'V8SI' is not supported for enumeration types}}
+  typedef T __attribute__((mode(V8SI))) T3; // expected-error2{{mode 'V8SI' is not supported for enumeration types}}
```

Comment:
This looks like it should just cause the expected check to go away?  What is going on here?  If this is causing it to happen 2x, we might wonder if the attribute `mode` should have not been generated if it errored the 1st time.